db/database.go

```diff
@@ -132,6 +134,7 @@ type DatabaseContext struct {
 	MetadataKeys                 *base.MetadataKeys             // Factory to generate metadata document keys
 	RequireResync                base.ScopeAndCollectionNames   // Collections requiring resync before database can go online
 	CORS                         *auth.CORSConfig               // CORS configuration
+	BlipStatsReportingInterval   int64                          // interval to report blip stats in milliseconds
```

Comment:
This feels like it should live in DatabaseContextOptions, what do you think?

Reply:
Added this to context options.